### ISSUE TITLE
Trouble with GRiSP’s robot demo (rebar3 fails)

### ISSUE THREAD

**Issue by dagyu** [grisp/rebar3_grisp#70]
Hi all, I have some problems when I try to deploy the robot example. 

`rebar3 grisp deploy` fails with this error `Config value for system_libs must be a boolean or directory but found: "/home/dagyu95/Documents/uni/tesi/robot/_grisp/grisp2/otp/23.3.4.11/install/lib"`.

I just followed the [wiki](https://github.com/grisp/grisp/wiki) and I don't know how to figure out the problem, can you give me some help?

### Verbose output

```
$ DIAGNOSTIC=1 rebar3 grisp deploy                    
===> Load global config file /home/dagyu95/.config/rebar3/rebar.config
===> 23.3.4.11 satisfies the requirement for minimum OTP version 18
===> Evaluating config script "/home/dagyu95/.cache/rebar3/plugins/hex_core/rebar.config.script"
===> 23.3.4.11 satisfies the requirement for minimum OTP version 19.3
===> Setting paths to [deps]
===> Compile (apps)
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Setting paths to [plugins]
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Compile (apps)
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Setting paths to [plugins]
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Compile (apps)
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Setting paths to [plugins]
===> Setting paths to [plugins]
===> Not adding provider grisp deploy from module rebar3_grisp_deploy because it already exists from module rebar3_grisp_deploy
===> Not adding provider grisp build from module rebar3_grisp_build because it already exists from module rebar3_grisp_build
===> Not adding provider grisp package from module rebar3_grisp_package because it already exists from module rebar3_grisp_package
===> Not adding provider grisp version from module rebar3_grisp_version because it already exists from module rebar3_grisp_version
===> Expanded command sequence to be run: []
===> Running provider: do
===> Expanded command sequence to be run: [app_discovery,install_deps,lock,compile,{grisp,deploy}]
===> Running provider: app_discovery
===> Found top-level apps: [robot]
	using config: [{src_dirs,["src"]},{lib_dirs,["apps/*","lib/*","."]}]
===> Evaluating config script "/home/dagyu95/Documents/uni/tesi/robot/_build/default/lib/grisp/rebar.config.script"
===> Setting paths to [deps]
===> Compile (apps)
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Setting paths to [plugins]
===> Setting paths to [plugins]
===> Not adding provider grisp deploy from module rebar3_grisp_deploy because it already exists from module rebar3_grisp_deploy
===> Not adding provider grisp build from module rebar3_grisp_build because it already exists from module rebar3_grisp_build
===> Not adding provider grisp package from module rebar3_grisp_package because it already exists from module rebar3_grisp_package
===> Not adding provider grisp version from module rebar3_grisp_version because it already exists from module rebar3_grisp_version
===> Running provider: install_deps
===> Verifying dependencies...
===> Running provider: lock
===> Running provider: compile
===> Setting paths to [deps]
===> Compile (apps)
===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Running hooks for compile with configuration:
===> 	{pre_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", pre_hooks, compile) -> no hooks defined

===> Compile (project_apps)
===> Running hooks for compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{pre_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", pre_hooks, compile) -> no hooks defined

===> Running hooks for erlc_compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{pre_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", pre_hooks, erlc_compile) -> no hooks defined

===> Setting paths to [deps]
===> Analyzing applications...
===> Compiling robot
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/home/dagyu95/Documents/uni/tesi/robot/src/robot_sup.erl",
                              "/home/dagyu95/Documents/uni/tesi/robot/src/robot.erl"]
===> Starting 0 worker(s)
===> Running hooks for erlc_compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{post_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", post_hooks, erlc_compile) -> no hooks defined

===> Running hooks for app_compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{pre_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", pre_hooks, app_compile) -> no hooks defined

===> Setting paths to [plugins]
===> Setting paths to [deps]
===> Running hooks for app_compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{post_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", post_hooks, app_compile) -> no hooks defined

===> Running hooks for compile in app robot (/home/dagyu95/Documents/uni/tesi/robot) with configuration:
===> 	{post_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", post_hooks, compile) -> no hooks defined

===> Running hooks for compile with configuration:
===> 	{post_hooks, []}.
===> run_hooks("/home/dagyu95/Documents/uni/tesi/robot", post_hooks, compile) -> no hooks defined

===> Setting paths to [plugins]
===> Running provider: {grisp,deploy}
===> [rebar3_grisp] [config]
===> [rebar3_grisp] [deploy]
===> [rebar3_grisp] [deploy,validate]
===> [rebar3_grisp] [deploy,validate,apps]
* Resolving OTP version
    23.3.4.11 (requirement was "23.3.4.11")
===> [rebar3_grisp] [deploy,collect]
===> GRiSP hash:
a80bccc41bebe9f21b8623d29b0c801277928295ab1f81531e37b2eb5353521e

[{<<"00100-rtems.patch">>,
  <<"987cab9e857a1ae86bfc6d12f17e1a50d6f1633a8f77d3da4cf99aa8c5badc28">>},
 {<<"00300-drivers-nifs.patch">>,
  <<"d3e469631ef8d253368f67d5c5d2b47ddf8290e48e7331362ca84863ec97b66">>},
 {<<"erts/emulator/drivers/unix/grisp_termios_drv.c">>,
  <<"7681801fca2aa5639e32365e9fad1626e3226f205bd3a7f4944e6e57ea29a3d0">>},
 {<<"erts/emulator/nifs/common/grisp_gpio_nif.c">>,
  <<"6d88dab1b11f41b7ca34791b67bb3dcdc288262db0c4c0ac01707248ed75194e">>},
 {<<"erts/emulator/nifs/common/grisp_hw_nif.c">>,
  <<"7be1ac99d35234c7370728d1bc3a1bdf6a31f3bca2c181406336959567e6c528">>},
 {<<"erts/emulator/nifs/common/grisp_i2c_nif.c">>,
  <<"eb989c69d1455eb84a9e3f0ff5646e4b9696cd8910a8f9f57d183925417c2cd7">>},
 {<<"erts/emulator/nifs/common/grisp_rtems_nif.c">>,
  <<"86840705643b73bda6edafaa6a65653346eede3f6be47c160842c192b6e98646">>},
 {<<"erts/emulator/nifs/common/grisp_spi_nif.c">>,
  <<"71af657167e7fe8b94463a589f0ee1a72e6572d5f41b28f685481cfcef0949d7">>},
 {<<"erts/emulator/sys/unix/erl_main.c">>,
  <<"bc5adb4b1518b6f29ea41cd3f6cf1e69919f5448cafe7f4db8c2866fad2bba4e">>},
 {<<"post-install-00500-clean.sh">>,
  <<"8257667f99ce2814c6f15f3b9b1c35765f986e9da1661736402dc91f78ad52a">>},
 {<<"xcomp/erl-xcomp-arm-rtems5.conf">>,
  <<"64db9be368d2598210da2d8eae558fde078e3b3814feb06eaed5c8981d4f392c">>}]
===> [rebar3_grisp] [deploy,package]
* Downloading pre-built OTP package (a80bccc4)
===> [rebar3_grisp] [deploy,package,meta]
===> [rebar3_grisp] [deploy,package,meta,package,
                            {meta,#{etag =>
                                        "\"96882f21787f2c89e3cc9fc6ddb4329e-4\""}}]
===> [rebar3_grisp] [deploy,package,init]
===> [rebar3_grisp] [deploy,package,init,
                            {deleting_tmp_file,
                                <<"/home/dagyu95/.cache/grisp/packages/otp/grisp_otp_build_23.3.4.11_a80bccc41bebe9f21b8623d29b0c801277928295ab1f81531e37b2eb5353521e.tar.gz.temp">>}]
===> [rebar3_grisp] [deploy,package,download]
===> [rebar3_grisp] [deploy,package,download,
                            {uri,"https://s3.amazonaws.com/grisp/platforms/grisp2/otp/grisp_otp_build_23.3.4.11_a80bccc41bebe9f21b8623d29b0c801277928295ab1f81531e37b2eb5353521e.tar.gz"}]
    (file cached)
* Extracting package
    (already extracted)
===> [rebar3_grisp] [deploy,release]
===> [rebar3_grisp] [deploy,release,
                            {start,#{erts =>
                                         <<"/home/dagyu95/Documents/uni/tesi/robot/_grisp/grisp2/otp/23.3.4.11/install">>,
                                     name => "robot",version => "0.1.0"}}]
===> ARGS: ["-n","robot","-v","0.1.0"]
===> ROOT: <<"/home/dagyu95/Documents/uni/tesi/robot/_grisp/grisp2/otp/23.3.4.11/install">>
===> Configuring releases the {relx, ...} entry from rebar.config
===> Config value for system_libs must be a boolean or directory but found: "/home/dagyu95/Documents/uni/tesi/robot/_grisp/grisp2/otp/23.3.4.11/install/lib"
```

### Versions

OTP: 23.3.4.11
rebar 3.18.0 on Erlang/OTP 23 Erts 11.2.2.10
rebar3_grisp (2.2.1)

**Comment by GalaxyGorilla** [comment]
See #69.

**Comment by GalaxyGorilla** [comment]
Fixed by v2.2.2.
